presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisMetadata.java

```diff
@@ -160,7 +160,7 @@ public ColumnMetadata getColumnMetadata(ConnectorSession connectorSession, Conne
             tableNames = ImmutableList.of(new SchemaTableName(prefix.getSchema().get(), prefix.getTable().get()));
         }
         else {
-            tableNames = listTables(session, null);
+            tableNames = listTables(session, Optional.empty());
```

Comment:
The call with `null` (as it used to be) should fail with NPE.
I hope https://github.com/prestosql/presto/issues/4293 would catch this.